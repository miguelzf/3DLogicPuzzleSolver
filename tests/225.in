- X B X - X
- - - - - -
- - X - X C
- - P - - -
- X R X - X
- - - - - L

- - X - X -
- - - - - P
- X - X - X
- - - R - -
- - X - X -
- - - - - -

- - - - - -
- X - X - X
C - - - - -
- L X B X -
- - - - - -
- X - X - X

