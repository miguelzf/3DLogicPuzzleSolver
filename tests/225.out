
Solution:
-  X  B  X  -  X  
b  b  b  -  -  -  
b  -  X  -  X  C  
b  -  P  p  p  p  
b  X  R  X  -  X  
b  r  r  l  l  L  
================
b  r  X  l  X  p  ||  p  p  p  c  c  -  
b  r  r  l  l  P  ||  -  X  -  X  c  X  
b  X  r  X  l  X  ||  C  c  c  c  c  -  
b  -  r  R  l  l  ||  l  L  X  B  X  -  
b  -  X  -  X  -  ||  b  b  b  b  -  -  
b  b  b  b  b  b  ||  b  X  -  X  -  X  
